src/lib/network/edit_nic_name.rb

```diff
@@ -137,8 +137,8 @@ def close
     #
     # Pops up an explanation if the name is invalid
     #
-    # @return [boolean] false if name is invalid
-    def CheckUdevNicName(name)
+    # @return [Boolean] false if name is invalid
+    def check_new_device_name(name)
```

Comment:
hmm, I am not fan of renaming in api proposal. Other documentation improvements :+1: